web/gds-user-ui/src/modules/dashboard/member/components/MemberModal/Copy.tsx

```diff
@@ -0,0 +1,110 @@
+import { useEffect, useState } from "react";
+import { copyToClipboard } from "../../utils";
+import { Button } from "@chakra-ui/react";
+import { Trans, t } from "@lingui/macro";
+import { getBusinessCategoryValue, getVaspCategoryValue } from "constants/basic-details";
+
+interface MemberDetailTableHeader {
+  label: string;
+  value: string;
+}
+
+type CopyProps = {
+    data: any;
+};
+
+function Copy({ data }: CopyProps) {
+const memberDetailTableHeader = [
```

Comment:
While trying to access the values for the export, I realized I was practically doing the same as we thought exporting would avoid. So, I decided to make the copy function work.